### ISSUE TITLE
104.31.67.210 (CloudFlare) недоступен

### ISSUE THREAD

**Issue by podivilov** [podivilov/runet-blackout#1]

**Comment by podivilov** [comment]
![](http://imgur.com/swgm8SVl.png)

**Comment by uprt** [comment]
В таких случаях, наверное, имеет смысл делать еще icmp- и udp-трейсроут, чтобы примерно понимать, на каком именно этапе возникают проблемы со связностью.

**Comment by podivilov** [comment]
@uprt, очевидно, что на шлюзе опсоса, где стоит DPI.

![](http://imgur.com/f8LNXqsl.png)

**Comment by podivilov** [comment]
@uprt, очевидно, что на шлюзе провайдера, у которого стоит DPI.

![http://imgur.com/f8LNXqsl.png]()

**Comment by f14a2cd** [comment]
Прошу прощения за оффтоп, но РКН решил сначала забанить сайт, потом IP, потом снова IP, потом вообще страницу. Это п*здец, товарищи!
![image](https://user-images.githubusercontent.com/57997247/71360368-788a7380-25b1-11ea-99cb-8f32ccaa0371.png)

**Comment by redbull05689** [comment]
Ну кто-то вместо одной палки срубил две))
